docs/contributing/plugins/README.md

```diff
@@ -28,26 +29,62 @@ basicPluginBuilder(
     # You can add more names after the inner column if multiple plugins use
     # this layout
     ('My plugin name',),
-    # A list of parameter indexes
-    [45, 46, 35, 37, 218, 219, 220, 221],
-    # A color to represent the parameters (this can also be a list)
-    Color.fromInteger(0x206cc8)
+    # A list of parameter indexes - these will map left to write to the 
+    # relevant faders on your device, so in this example tuning will be
+    # fader 1, Waveform will be fader 2, Cutoff will be Fader 3, Resonance
+    # will be Fader 4 and so on.
+    [
+    0,  # Tuning
+    1,  # Waveform
+    2,  # Cutoff
+    4,  # Resonance
+    5,  # Envelope Modulation
+    6,  # Decay
+    7,  # Accent
+    8,  # Volume
+    ]
+    # A color to represent the parameters (this can also be a list, to do so you
+    # would need to list the colour parameters in square brackets with commas between
+    # each parameter - ie ([0x206cc8,0x222222,0x888888,0x206cc8,0x222222])
```

Comment:
These would each need to be converted into colour objects. Each value should be wrapped in a `Color.fromInteger(...)` call.